docker/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '2'
+services:
+
+  # The leastauthority.com website, including handling of the signup forms.
+  web:
+    # See build.sh
+    image: 'leastauthority.com/web'
+
+    ports:
+      # The web server in the container listens for HTTPS on 8443.
+      # Publish that to the world on the standard HTTPS port.
+      - '443:8443'
+
+    volumes:
+
+      # The web server generates a lot of random output right now.
+      # Subscription logs, server logs, etc.  Give it a safe place to
+      # dump all of that data.
+      - 'web-data:/app/data'
+
+      # This is unfortunate.  It's here just so we can get the signup
+      # furl generated at flappserver startup time.  The web server
+      # reads the signup.furl file that the flapp server writes into
+      # this volume.
+      - 'flapp-data:/app/flapp-data'
+
+    depends_on:
+
+      # Start the flapp server first so the signup.furl is written.
+      # This is subject to startup race-conditions since Docker has no
+      # idea when signup.furl is written: it only knows that it has
+      # _started_ the flapp container.  Experiments suggest the flapp
+      # server typically wins.  Ideally some alternate mechanism could
+      # be used to convey the signup furl between processes (in other
+      # words, fix the application to not use the filesystem for RPC).
+        - "flapp"
```

Comment:
I don't know why flappserver was used, but I strongly suspect it was along the lines of "we're already using Foolscap, let's use *moar foolscap*" or similar ;) Maybe @daira knows the answer.